html/pages/device.inc.php

```diff
@@ -173,6 +173,14 @@
                 </li>';
         }
 
+        if (@dbFetchCell("SELECT COUNT(id) FROM tnmsneinfo WHERE device_id = '".$device['device_id']."'") > '0') {
```

Comment:
Check the os before running a query.  Like this: `if ($device['os'] == 'coriant' &&` here